GameData/RP-1/Contracts/Earth Observation 1/EOSIRRad1Rep.cfg

```diff
@@ -0,0 +1,160 @@
+	
+	description = <b>Program: Early Earth Observation Satellites<br>Type: <color=blue>Optional</color></b><br><br>After the success of your first weather satellite, the US Weather Bureau wants additional satellites available for more timely weather observations with better coverage of Earth.  Launch additional weather satellites to fulfill the Weather Bureau's request.<br><br><b><color=yellow>NOTE: The satellite will be destroyed upon completion of the contract. This simulates transfer of satellite control to the customer.</color></b><br><br><b><color="green">Make sure your craft contains an Infrared Radiometry 1 experiment and an Early TV camera.</color></b><br><br><b>Number of Contracts Completed: @index / @maxCompletions</b><br><br>Real life examples: TIROS-2 through TIROS-8.
+
+	synopsis = Launch additional weather satellites with the Infrared Radiometry experiment and an Early TV camera.
+
+	completedMessage = Congratulations on another succesful launch.  Our use of satellites to better understand the impacts of weather phenomena and improve forecasting capabilities continues to improve with every launch.
+
+	sortKey = 402
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 7
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 20
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyEOS
+	}
+
+	REQUIREMENT
+ 	{
+  		name = CompleteContract
+    		type = CompleteContract
+      		contractType = ScienceSatIRRad1
+		title = Complete first IR Radiometry satellite.
+	}
+	
+	DATA
+	{
+		type = int
+		index = $RepeatIRRadSats_Count + 0
+	}
+
+	BEHAVIOUR
+	{
+		name = IncrementTheCount
+		type = Expression
+		
+		CONTRACT_COMPLETED_SUCCESS
+		{
+			RepeatIRRadSats_Count = $RepeatIRRadSats_Count + 1
+		}
+	}
+	
+	PARAMETER
+	{
+		name = ScienceSatIRRad1Repeat
+		type = VesselParameterGroup
+		title = Additional weather satellite
+		define = ScienceSatIRRad1Repeat
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+
+		PARAMETER
+		{
+			name = OrbitSequence
+			type = Sequence
+			title = Survive in orbit for a day and transmit science
+
+			PARAMETER
+			{
+				name = HasPower
+				type = HasResource
+				resource = ElectricCharge
+				minQuantity = 1.0
+				title = Craft must have ElectricCharge after one day
+				hideChildren = true
+			}
+		}
+		PARAMETER
+		{
+			name = Orbit
+			type = Orbit
+			minInclination = Round(Random(30, 80))

```

Comment:
This probably should just be based on completing IR Rad in space low? Otherwise it's just "launch a thing with no payload to a random orbit" which doesn't feel like it fits the new program. If it's just "get max 80%" of IR rad though then there's only one go at this contract, so not sure what else to add.